### DISCUSSION TITLE
JavaFxMediaRegistry cannot be resolved

### DISCUSSION THREAD

**Discussion by InkwellIdeas** [webfx-project/webfx#19] [Q&A] [answered]
Hi Folks,
I love the goal you all have, and when I tested some samples our a month or two ago things were looking promising but not featured enough.  Now when I try the files sample project I get this when I do webfx build --gwt  (--openjfx-fatjar works):
[INFO] --- gwt-maven-plugin:1.0.1:compile (default-compile) @ webfx-demo-files-application-gwt ---
[INFO] Compiling module webfx-demo-files-application-gwt
[INFO]    Tracing compile failure path for type 'javafx.scene.media.MediaView'
[INFO]       [ERROR] Errors in 'jar:file:/C:/Users/joewe/.m2/repository/dev/webfx/webfx-kit-javafxmedia-emul/0.1.0-SNAPSHOT/webfx-kit-javafxmedia-emul-0.1.0-SNAPSHOT-sources.jar!/javafx/scene/media/MediaView.java'
[INFO]          [ERROR] Line 33: JavaFxMediaRegistry cannot be resolved
[INFO]          [ERROR] Line 6: The import dev.webfx.kit.registry.javafxmedia cannot be resolved
[INFO]    [ERROR] Aborting compile due to errors in some input files

 Any ideas to help me zero in on the issue?

**Comment by salmonb** [comment, marked as answer]
Thanks @InkwellIdeas for reporting that.

The working version was in a separate branch that I forgot to merge into main. It's now done, and it should solve your issue. And this new version supports video files as opposed to the previous one.

**Reply by salmonb** [reply]
Discussion moved to issue https://github.com/webfx-project/webfx/issues/28 (now closed)
